Add a readme (#1)
DevExpress-Examples/devextreme-datagrid-modify-persisted-state: README.md

<!-- default badges list -->
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# DevExtreme DataGrid - How to modify the persisted state

> This example shows how to exclude component settings that you do not want restored after the page is reloaded.

Reviewer: ```suggestion
This example shows how to exclude items from component settings that are restored after the page is reloaded.
```

> This example shows how to exclude component settings that you do not want to be restored after the page is reloaded.

Reviewer 2: ```suggestion
This example shows how to exclude component settings that you don't need restored after the page is reloaded.
```

> In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all the modifications persist, except filters applied to the grid columns.

Reviewer: ```suggestion
In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all the modifications persist, except filters applied to grid columns.
```

> In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all the modifications persist, except filters applied to grid columns.

Reviewer 2: ```suggestion
In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all modifications will persist, except filters applied to grid columns.
```

> The demonstrated approach can be used with the following UI components:

Reviewer: ```suggestion
You can use the demonstrated approach with the following UI components:
```
